### PR TITLE
Add CIPHER (CPR)

### PR DESCRIPTION
**Pull request by cphertech**
Add our token CIPHER (CPR)

### CODE REVIEW

**Comment by cphertech** [comment]
Hello, Polygon Dev. Team, Kindly add CIPHER (CPR) in the verified token list. Thank you.

File: src/tokens/polygonTokens.json [hunk 1]
```diff
@@ -20519,4 +20519,22 @@
             }
         }
     }
```
Comment (py-zoid) [inline src/tokens/polygonTokens.json:20521]: Missing a comma "," after the previous entry which is breaking the build process. Also check indentation. you can check for errors running "npm run lint".

**Review by py-zoid — CHANGES_REQUESTED** [review]

File: src/tokens/polygonTokens.json (hunk 1 above)
Comment (cphertech) [reply src/tokens/polygonTokens.json:20521]: Hi, updated the error with new pull request, please check and do the needful. Thank you.

**Comment by py-zoid** [comment]
Duplicated in PR #318. Closing this.

**Comment by cphertech** [comment]
Dear Rajesh
Good afternoon. We have rectified the error. Kindly check the new PR link
https://github.com/maticnetwork/polygon-token-list/pull/321   Kindly do the
needful. Thank you.
Best regards,
Raja

**Pull request closed without merging**